src/qgismobileapp.cpp

```diff
@@ -268,6 +278,36 @@ void QgisMobileapp::loadProjectFile( const QString& path )
   emit loadProjectEnded();
 }
 
+void QgisMobileapp::print( int layoutIndex )
+{
+  const QList<QgsPrintLayout *> projectLayouts( mProject->layoutManager()->printLayouts() );
+
+  const auto &layoutToPrint = projectLayouts.at( layoutIndex );
```

Comment:
This will crash if the index is out of bounds. Not sure if you caught that somewhere else